apps/meteor/ee/client/voip/modal/DialPad/hooks/useLongPress.tsx

```diff
@@ -0,0 +1,63 @@
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+import { useCallback, useRef } from 'react';
+
+type UseLongPressResult = {
+	onClick: () => void;
+	onMouseDown: () => void;
+	onMouseUp: () => void;
+	onTouchStart: () => void;
+	onTouchEnd: () => void;
+};
+
+export function useLongPress(onLongPress: () => void, options?: { onClick?: () => void; threshold?: number }): UseLongPressResult {
+	const action = useRef<'LONG_PRESS' | 'CLICK'>();
+	const isLongPress = useRef(false);
+	const timerRef = useRef<NodeJS.Timeout>();
+
+	const startPressTimer = useMutableCallback((): void => {
+		isLongPress.current = false;
+		timerRef.current = setTimeout(() => {
+			isLongPress.current = true;
+			action.current = 'LONG_PRESS';
```

Comment:
`isLongPress` and `action.current === 'LONG_PRESS'` seem to store the same information. Can we remove one of them?

Reply:
No, because of handleOnClick control... I think both are necessary on this case 😞

Reply:
No, you are correct... thanks!